### PR TITLE
Fix Intl error android

### LINKED ISSUE #2521: Intl TypeError on android devices
The root cause of this is due to Intl being undeclared resulting in a ReferenceError

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #2521

loki-ooph

### CODE REVIEW

File: src/core/i18n/utils.js [hunk 1]
```diff
@@ -246,7 +246,7 @@ export function makeI18n(
   i18nData: I18nConfig,
   lang: string,
   _Jed: Jed = Jed,
-  { _Intl = Intl }: makeI18nOptions = {}
+  { _Intl = window.Intl }: makeI18nOptions = {}
```
Comment (kumar303) [inline src/core/i18n/utils.js]: This is [used in server code](https://github.com/mozilla/addons-frontend/blob/master/src/core/server/base.js#L32) so it needs to be protected by `if (typeof window !== 'undefined')...` or something like that.

Comment (muffinresearch) [inline src/core/i18n/utils.js]: Ah yep you're right. 🤕

File: src/core/i18n/utils.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* global Intl */
+/* global Intl, window */
```
Comment (kumar303) [inline src/core/i18n/utils.js]: You don't need to declare this as a global anymore

**Comment by kumar303** [comment]
bah, looks like https://github.com/facebook/flow/issues/4077

I suggest adding a [`$FLOW_FIXME`](https://github.com/mozilla/addons-frontend/blob/master/.flowconfig#L28) comment which is different than an ignore comment in that it's something we should be able to search for and fix later.

**Pull request merged**